packages/kit/src/runtime/client/utils.js

```diff
@@ -98,7 +98,7 @@ export function find_anchor(element, base) {
 	/** @type {typeof valid_link_options['reload'][number] | null} */
 	let reload = null;
 
-	while (element !== document.documentElement) {
+	while (element && element !== document.documentElement) {
```

Comment:
do you really need to make this change? are you getting `null` or `undefined` for `documentElement`?

Reply:
no, but if element is `null`, it wouldn't stop at this point